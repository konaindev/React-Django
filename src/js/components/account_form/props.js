export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
