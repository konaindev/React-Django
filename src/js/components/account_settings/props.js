const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
