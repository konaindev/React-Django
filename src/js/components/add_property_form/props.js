export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
