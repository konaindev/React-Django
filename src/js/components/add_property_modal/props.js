import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
