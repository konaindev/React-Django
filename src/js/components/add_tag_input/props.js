const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
