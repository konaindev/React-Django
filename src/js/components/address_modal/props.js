export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
