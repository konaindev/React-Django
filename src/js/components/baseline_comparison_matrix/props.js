import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
