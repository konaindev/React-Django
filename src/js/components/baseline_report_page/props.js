import { project, report_links } from "../project_page/props";
import { funnel_history } from "../funnel_performance_analysis/FunnelProps";

export const report = {
  dates: {
    start: "2017-07-24",
    end: "2018-07-23"
  },
  property_name: "Portland Multi-Family",
  property: {
    average_monthly_rent: "1847.00",
    lowest_monthly_rent: "1847.00",
    cost_per_exe_vs_rent: 0.54,
    total_units: 201,
    leasing: {
      change: 36,
      cds: 28,
      cd_rate: 0.29,
      renewal_notices: 94,
      renewals: 94,
      renewal_rate: 0.71,
      resident_decisions: 132,
      vacation_notices: 38,
      rate: 0.74,
      units: 192
    },
    occupancy: {
      move_ins: 72,
      move_outs: 36,
      rate: 0.71,
      units: 185,
      occupiable: 260
    }
  },
  funnel: {
    volumes: {
      usv: 19621,
      inq: 785,
      tou: 259,
      app: 96,
      exe: 68
    },
    costs: {
      usv: "3.47",
      inq: "86.62",
      tou: "262.00",
      app: "708.00",
      exe: "1000.00"
    },
    conversions: {
      usv_inq: 0.04,
      inq_tou: 0.33,
      tou_app: 0.37,
      app_exe: 0.71,
      usv_exe: 0.003
    }
  },
  investment: {
    acquisition: {
      expenses: {
        demand_creation: "48000.00",
        leasing_enablement: "0.00",
        market_intelligence: "0.00",
        reputation_building: "20000.00"
      },
      total: "68000.00",
      romi: 10,
      estimated_revenue_gain: "709200.00"
    },
    retention: {
      expenses: {
        demand_creation: "0.00",
        leasing_enablement: "6000.00",
        market_intelligence: "0.00",
        reputation_building: "0.00"
      },
      total: "6000.00",
      romi: 346,
      estimated_revenue_gain: "2080000.00"
    },
    total: {
      total: "74000.00",
      romi: 38,
      estimated_revenue_gain: "2800000.00"
    }
  },
  four_week_funnel_averages: {
    usv: 1509,
    inq: 60,
    tou: 20,
    app: 7,
    exe: 5
  },
  funnel_history
};

const current_report_link = report_links.baseline;

const share_info = {
  shared: true,
  share_url: `/projects/${project.public_id}/share/baseline/`,
  update_action: "shared_reports"
};

export default {
  project,
  report,
  report_links,
  current_report_link,
  share_info
};
