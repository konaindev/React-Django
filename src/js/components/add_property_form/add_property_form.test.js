import React from "react";
import renderer from "react-test-renderer";

import AddPropertyForm from "./index";
import { props } from "./props";

describe("AddPropertyForm", () => {
  it("add property form", () => {
    const tree = renderer.create(<AddPropertyForm {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
