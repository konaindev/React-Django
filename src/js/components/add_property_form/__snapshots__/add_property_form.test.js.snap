// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddPropertyForm add property form 1`] = `
<form
  action={null}
  autoComplete="off"
  className="add-property-form"
  method="post"
  onReset={[Function]}
  onSubmit={[Function]}
>
  <div
    className="add-property-form__column"
  >
    <div
      className="add-property-form__image"
      style={Object {}}
    >
      <label
        className="add-property-form__file"
      >
        ADD PHOTO
        <input
          accept="image/*"
          name="building_photo"
          onChange={[Function]}
          style={
            Object {
              "display": "none",
            }
          }
          type="file"
        />
      </label>
    </div>
  </div>
  <div
    className="add-property-form__column"
  >
    <div
      className="add-property-form__field"
    >
      <div
        className="add-property-form__label"
      >
        Property Name:
      </div>
      <div>
        <input
          className="input add-property-form__input"
          name="property_name"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="Property Name"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="add-property-form__field"
    >
      <div
        className="add-property-form__label"
      >
        Address:
      </div>
      <div>
        <input
          className="input add-property-form__input"
          name="street_address_1"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="Street Address 1"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="add-property-form__field"
    >
      <div
        className="add-property-form__label"
      />
      <div>
        <input
          className="input add-property-form__input"
          name="street_address_2"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="Street Address 2"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="add-property-form__field"
    >
      <div
        className="add-property-form__label"
      />
      <div>
        <div
          className="add-property-form__fields-wrap"
        >
          <input
            className="input add-property-form__input add-property-form__input--city"
            name="city"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="City"
            type="text"
            value=""
          />
          <div
            className="select add-property-form__input add-property-form__input--state css-2b097c-container"
            onKeyDown={[Function]}
          >
            <div
              className="select__control css-yk16xz-control"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
            >
              <div
                className="select__value-container css-1hwfws3"
              >
                <div
                  className="select__placeholder css-1wa3eu0-placeholder"
                >
                  State
                </div>
                <input
                  className="css-62g3xt-dummyInput"
                  disabled={false}
                  id="react-select-2-input"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  readOnly={true}
                  tabIndex="0"
                  value=""
                />
              </div>
              <div
                className="select__indicators css-1wy0on6"
              >
                <span
                  className="select__indicator-separator css-1okebmr-indicatorSeparator"
                />
                <div
                  aria-hidden="true"
                  className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <svg
                    aria-hidden="true"
                    className="select__dropdown-arrow select__dropdown-arrow--down"
                    color="inherit"
                    focusable="false"
                    height="1em"
                    role="presentation"
                    viewBox="0 0 10 5"
                    width="1em"
                  >
                    <path
                      d="M1 1.16134L4.92334 4.5"
                      stroke="currentColor"
                      strokeLinecap="round"
                    />
                    <path
                      d="M8.82812 1.16134L5 4.5"
                      stroke="currentColor"
                      strokeLinecap="round"
                    />
                  </svg>
                  <svg
                    aria-hidden="true"
                    className="select__dropdown-arrow select__dropdown-arrow--up"
                    color="inherit"
                    focusable="false"
                    height="1em"
                    role="presentation"
                    viewBox="0 0 10 5"
                    width="1em"
                  >
                    <path
                      d="M1 3.83866L4.92334 0.5"
                      stroke="currentColor"
                      strokeLinecap="round"
                    />
                    <path
                      d="M8.82812 3.83866L5 0.5"
                      stroke="currentColor"
                      strokeLinecap="round"
                    />
                  </svg>
                </div>
              </div>
            </div>
            <input
              name="state"
              type="hidden"
              value=""
            />
          </div>
          <input
            className="input add-property-form__input add-property-form__input--zipcode"
            name="zip_code"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Zipcode"
            type="text"
            value=""
          />
        </div>
      </div>
    </div>
    <div
      className="add-property-form__field"
    >
      <div
        className="add-property-form__label"
      >
        Package Option:
      </div>
      <div>
        <div
          className="select add-property-form__input css-2b097c-container"
          onKeyDown={[Function]}
        >
          <div
            className="select__control css-yk16xz-control"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <div
              className="select__value-container css-1hwfws3"
            >
              <div
                className="select__placeholder css-1wa3eu0-placeholder"
              >
                Select a Package...
              </div>
              <input
                className="css-62g3xt-dummyInput"
                disabled={false}
                id="react-select-3-input"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                readOnly={true}
                tabIndex="0"
                value=""
              />
            </div>
            <div
              className="select__indicators css-1wy0on6"
            >
              <span
                className="select__indicator-separator css-1okebmr-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
                onMouseDown={[Function]}
                onTouchEnd={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="select__dropdown-arrow select__dropdown-arrow--down"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 10 5"
                  width="1em"
                >
                  <path
                    d="M1 1.16134L4.92334 4.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                  <path
                    d="M8.82812 1.16134L5 4.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  className="select__dropdown-arrow select__dropdown-arrow--up"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 10 5"
                  width="1em"
                >
                  <path
                    d="M1 3.83866L4.92334 0.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                  <path
                    d="M8.82812 3.83866L5 0.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                </svg>
              </div>
            </div>
          </div>
          <input
            name="product_type"
            type="hidden"
            value=""
          />
        </div>
      </div>
    </div>
    <div
      className="add-property-form__field add-property-form__field--button"
    >
      <div
        className="add-property-form__label"
      />
      <div
        className="add-property-form__submit-wrap"
      >
        <button
          className="button button--disabled add-property-form__submit"
          onClick={[Function]}
          type="submit"
        >
          SUBMIT
        </button>
      </div>
    </div>
  </div>
</form>
`;
